### PR TITLE
Add support for the "--external-delegate-path" to the CMake build of the benchmark tool

### PR DESCRIPTION
**Pull request by NAgarwalla**
8f7b238: added support for the "--external-delegate-path" to the cmake build of the benchmark tool.  The support for external delegates is already part of the bazel build.

4a85ea5: Adds TFLITE_ENABLE_HEXAGON around code that should have it.

### CODE REVIEW

**Review by terryheo — APPROVED** [review]
LGTM

File: tensorflow/lite/CMakeLists.txt [hunk 1]
```diff
@@ -67,6 +67,7 @@ cmake_dependent_option(TFLITE_ENABLE_NNAPI_VERBOSE_VALIDATION "Enable NNAPI verb
 option(TFLITE_ENABLE_MMAP "Enable MMAP (unsupported on Windows)" ON)
 option(TFLITE_ENABLE_GPU "Enable GPU" OFF)
 option(TFLITE_ENABLE_XNNPACK "Enable XNNPACK backend" ON)
+option(TFLITE_ENABLE_EXTERNAL "Enable External Delegate backend" ON)
```
Comment (terryheo) [inline tensorflow/lite/CMakeLists.txt:70]: The name doesn't look trivial.
How about TFLITE_ENABLE_EXTERNAL_DELEGATE ?

Comment (NAgarwalla) [reply tensorflow/lite/CMakeLists.txt:70]: Sure, that works for me.  I was trying to follow the convention for the other similar delegate options.  I agree "EXTERNAL" by itself is vague.  I'll make the change and resubmit.

**Comment by NAgarwalla** [comment]
Sorry, apologize for closing this ticket, also. I'm still getting used to github.  I want to commit the two commits as two separate pull requests.  I'll be resubmitting (again).

**Pull request closed without merging**